src/auth/auth-config.ts

```diff
@@ -816,6 +897,8 @@ export class OIDCConfig implements OIDCAuthProviderConfig {
       providerId: this.providerId,
       issuer: this.issuer,
       clientId: this.clientId,
+      clientSecret: this.clientSecret,
+      responseType: this.responseType,
```

Comment:
I would copy the content here so developers cannot modify the content.
`responseType:  deepCopy(this.responseType),`